charts/k8s-watcher/templates/configmap.yaml

```diff
@@ -7,4 +7,5 @@ metadata:
   {{- end }}
 data:
   komodor-k8s-watcher.yaml: |
-{{- include "watcher.values" . | nindent 4 }}
\ No newline at end of file
+{{- include "watcher.values" . | nindent 4 }}
+    clusterName: {{ .Values.clusterName | required "Clustername is rquired" }}
```

Comment:
Our `clusterName` lives under `watcher` AFAIK. Is there a mistake to use `.Values.clusterName`?

Reply:
Also a misprint there in `Clustername is rquired` and also `Clustername`!=`clusterName`

Reply:
whops for both , good catch :)